app/views/passengers/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<h1>Passenger <%= @passenger.id %>: <%= @passenger.name %></h1>
+<p>
+  <strong>ID:</strong>
+  <%= @passenger.id %>
+  <br />
+  <strong>Passenger:</strong>
+  <%= @passenger.name %>
+  <br />
+  <strong>Phone Number:</strong>
+  <%= @passenger.phone_num %>
+  <br />
+  <strong>Money spent:</strong>
+  $<%= @passenger.spent %>
+</p>
+
+<%= form_for @trip, url: passenger_trips_path(@passenger.id) do |f| %>
+  <%= f.submit "Request New Trip", class: "submit-button" %>
+<% end %>
+
+<h2>Trips for this <%= @passenger.name %></h2>
+  <ul>
+    <% @passenger.trips.sort.each do |trip| %>
+    <p>
+      <li><strong>Trip: <%=trip.id %></strong></li>
+      <li>Date: <%=trip.date %></li>
+      <% if trip.cost == nil %>
+        <li>Cost: Ride in progress</li>
```

Comment:
Good choice! There wasn't a requirement about what to do with the trip's cost, so we asked you all to make a decision on it... I like this decision!